src/App.tsx

```diff
@@ -138,15 +138,31 @@ const SettingsComponent: React.FC = () => {
           </fieldset>
           <fieldset>
             <label>
-              Model:
+              Select Model:
               <br />
               <input
                 type="text"
-                value={defaultOpenAiModel}
```

Comment:
if we remove the value, the app won't display it when closing and opening the browser. Can you introduce it back?

The behavior, of not showing all the options once one option is typed is acceptable on inputs with auto-completion and will be already a huge improvement from the previous version.

Reply:
Re-added the default value e74b7fbf2bbb9ee2a438968d4aaf99fed6bb64a1